invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -73,3 +69,37 @@ body {
 .padded-small {
   padding: 1rem 0 !important;
 }
+
+// Export page
+pre.export.result {
+    display:block;
+    white-space: pre-wrap;
+    padding:9.5px;
+    margin:0 0 10px;
+    font-size:13px;
```

Comment:
do we use `px` for fonts? I thought that also @jennur was advocating for `em`/`rem`?
Well, apparently, there are [many opinions](https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css) on this :)

Reply:
I'm voting for em/rem, yes. Then it'll always scale with the browser settings. Also, semantic ui is already using em/rem.

Reply:
this is only copy paste of the code, I can change it in the next iteration, i agree we should use relative units